handbook/company.md

```diff
@@ -53,152 +53,152 @@ Empathy leads to better understanding, better communication, and better decision
-  - pick low-hanging fruit (deliver value quickly where you can)
-  - think ahead, then make the right decision for now
-  - look before you leap (when facing a non-trivial problem, get perspective before you dive in; what if there is a simpler solution?) <!-- TODO: When facing a (non-trivial) problem, take a step back before diving into fixing it - put the problem back in context, think about the actual goal and not just the issue itself, sometimes the obvious solution misses the end goal, sometimes a simpler solution will emerge, or it may just confirm that the fix is the right one and you can go ahead with better confidence -->
-- move quickly
-  - "everything is in draft"
-  - think, fast (balance thoughtfulness and planning with moving quickly)
-  - aim to deliver daily
-  - move quicker than 90% of the humans you know
-  - resist gold-plating; avoid bike-shedding
-- less is more
-  - focus on fewer tasks at one time  <!-- TODO: (By focusing on fewer tasks at once, we are able to get more done, and to a higher standard, while feeling more positive about our work in the process.) -->
-  - "boring solutions"
-  - finish what you start, or at least throw it away loudly in case someone wants it
-  - keep it simple (prioritize simplicity; people crave mental space in design, collaboration, and most areas of life) <!-- reduce cognitive load -->
-  - use fewer words (lots of text == lots of work)
-  - as time allows  ("I would have written a shorter letter, but I did not have the time." -Blaise Pascal)
-- make time for self-care
-  - to help you bring your best self when communicating with others, making decisions, etc
-  - consider taking a break or going for a walk
-  - take time off; it is better to have 100% focus for 80% of the time than it is to have 80% focus for 100% of the time
-  - think about how to best organize your day/work hours to fit your life and maximize your focus
-- better focus
+- Iterate your work.
+  - Work in baby steps. <!-- TODO: (look for ways to make the smallest, minimally viable change. Small changes provide faster feedback, and help us to stay focused on quality) -->
+  - Pick low-hanging fruit (deliver value quickly where you can).
+  - Think ahead, then make the right decision for now.
+  - Look before you leap (when facing a non-trivial problem, get perspective before you dive in; there may be a simpler solution). <!-- TODO: When facing a (non-trivial) problem, take a step back before diving into fixing it - put the problem back in context, think about the actual goal and not just the issue itself, sometimes the obvious solution misses the end goal, sometimes a simpler solution will emerge, or it may just confirm that the fix is the right one and you can go ahead with better confidence -->
+- Move quickly.
+  - "Everything is in draft."
+  - Think, fast so you can work quickly.
```

Comment:
```suggestion
  - Think fast (balance thoughtfulness and planning with moving quickly).
```

Reply:
I like that your suggestion is more punchy, but "balance thoughtfulness and planning with moving quickly" is important. Working quickly is good, but thinking and planning are also important - just don't get stuck in the weeds with planning.